docs/Welcome.md

```diff
@@ -67,3 +67,7 @@ By separating code into distinct _nodes_, different _nodes_ can be run in parall
 
 :::
 
+
+<br />
+Features of Ganymede interact with users' file system and are not compatible with Mozilla Firefox or Internet Explorer.
```

Comment:
Features of Ganymede that interact with users' file system are not compatible with Mozilla Firefox or Internet Explorer.